MITADS-Speech/corpora_importer.py

```diff
@@ -231,15 +245,30 @@ def _write_csv(self,corpus:Corpus):
         samples_len = len(audios)
         for _file in audios:
 
+            row_data = None
+            for r in filtered_rows:
+                if(r[0]==_file):
+                   row_data = r
+                   break 
+
+            ##skip if not filtered
+            if(row_data==None):
+                continue
+            
+            duration = row_data[3]
+            comments = row_data[4]
+
             st = os.stat(_file)
             file_size = st.st_size
             utterence = utterences[_file]
+
             utterence_clean = utterence      
+            speaker_id = corpus.get_speaker_id(_file)
```

Comment:
Where `get_speaker_id` is defined?

Reply:
sorry I forgot to add the implemented method i Corpus class, without this going into error. However it is not used in MLS importer yet.